node_modules/oae-core/documentpreview/documentpreview.html

```diff
@@ -52,9 +52,17 @@
 
 <!-- TEMPLATES -->
 <div id="documentpreview-content-page-template"><!--
-    <div class="documentpreview-content-page ${cssScopeClass}" data-page-number="${pageNumber}">
-        <div class="documentpreview-content-page-loading text-center">
-            <i class="fa fa-spinner fa-spin"><span class="sr-only">__MSG__LOADING__</span></i>
+    <div class="documentpreview-content-page" data-page-number="${pageNumber}">
+        <div class="documentpreview-content-text-page sr-only"">
+            <label for="documentpreview-content-text-page-${pageNumber}">
+                __MSG__PAGE_X_OF_Y_PREFIX__ ${pageNumber} __MSG__PAGE_X_OF_Y_MIDDLE__ ${totalPages} __MSG__PAGE_X_OF_Y_SUFFIX__
+            </label>
+            <textarea readonly id="documentpreview-content-text-page-${pageNumber}">__MSG__LOADING__</textarea>
```

Comment:
I haven't found any suggestions that this is a problem, and it works perfectly with VoiceOver. Using a `<textarea>` puts the page into the tab focus group, so users can tab from one page to the next. (It works for sighted users as well; tabbing scrolls the page into view.)